routes/api.js

```diff
@@ -2,21 +2,41 @@ const express = require ('express');
 const router = express.Router();
 const bcrypt=require('bcryptjs');
 const User = require('../models/user');
+const jwt = require('jsonwebtoken');
 
+router.post('/users/login',function(req,res,next){
+    User.findOne({email:req.body.email}).then(function(user){
+      if(user.length<1){
+        res.status(401).json({message:'Authentication failed'})
```

Comment:
User  not found instead of authentication failed

Reply:
@Kush223 plz fix it